docs/docs/references/metrics.mdx

```diff
@@ -14,7 +14,7 @@ In Lightdash, metrics are used to summarize dimensions or, sometimes, other metr
 There are two ways to add metrics to your project in Lightdash:
 
 1. (**Suggested**) [Using the `meta` tag](#using-the-meta-tag)
-2. [Using dbt's Semantic Layer](#using-dbts-semantic-layer)
+2. [Using dbt's Semantic Layer](#using-dbts-semantic-layer) (this is still a Beta feature)
```

Comment:
Can you update the text in parenthesis to: "(This integration is in public beta in Lightdash)"